collector/collector_linux_test.go

```diff
@@ -0,0 +1,41 @@
+package collector
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/coreos/go-systemd/journal"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSystemd_Collect(t *testing.T) {
+	if os.Getenv("TRAVIS") != "" {
+		t.Skipf("SKIPPING: We can not read from journal in Travis")
+	}
+	if !journal.Enabled() {
+		t.Skipf("SKIPPING: Journal not enabled")
```

Comment:
Hmm, shouldn't we fail the test here? How do we make sure that the test is actually run somewhere?

Reply:
https://github.com/dcos/dcos-diagnostics/pull/116